### PR TITLE
Change the default value of `feedback-probability`

### PR DESCRIPTION
**Pull request by winoros**
### What is changed, added or deleted? (Required)

`feedback-probability` 的默认值说明有误

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [x] Might cause conflicts

### CODE REVIEW

File: statistics.md [hunk 1]
```diff
@@ -108,7 +108,11 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
 
 当某个表 `tbl` 的修改行数与总行数的比值大于 `tidb_auto_analyze_ratio`，并且当前时间在 `tidb_auto_analyze_start_time` 和 `tidb_auto_analyze_end_time` 之间时，TiDB 会在后台执行 `ANALYZE TABLE tbl` 语句自动更新这个表的统计信息。
 
-在查询语句执行时，TiDB 会以 `feedback-probability` 的概率收集反馈信息，并将其用于更新直方图和 Count-Min Sketch。`feedback-probability` 可通过配置文件修改，其默认值是 `0.0`。
+在查询语句执行时，TiDB 会以 `feedback-probability` 的概率收集反馈信息，并将其用于更新直方图和 Count-Min Sketch。`feedback-probability` 可通过配置文件修改，其默认值是 `0.05`, 设置成 `0.0` 可以关闭这个功能。
+
+> **注意：**
+>
+> 在配置文件中如果将 `feedback-probability` 设置为 `0` 会导致设置失败报错。
```
Comment (SunRunAway) [inline statistics.md:115]:
```suggestion
> 在配置文件中如果将 `feedback-probability` 设置为 `0` 会导致设置失败报错，需要设置成 `0.0`，才可以将 `feedback-probability`关闭。
```

File: statistics.md [hunk 2]
```diff
@@ -108,7 +108,11 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
 
 当某个表 `tbl` 的修改行数与总行数的比值大于 `tidb_auto_analyze_ratio`，并且当前时间在 `tidb_auto_analyze_start_time` 和 `tidb_auto_analyze_end_time` 之间时，TiDB 会在后台执行 `ANALYZE TABLE tbl` 语句自动更新这个表的统计信息。
 
-在查询语句执行时，TiDB 会以 `feedback-probability` 的概率收集反馈信息，并将其用于更新直方图和 Count-Min Sketch。`feedback-probability` 可通过配置文件修改，其默认值是 `0.0`。
+在查询语句执行时，TiDB 会以 `feedback-probability` 的概率收集反馈信息，并将其用于更新直方图和 Count-Min Sketch。`feedback-probability` 可通过配置文件修改，其默认值是 `0.05`, 设置成 `0.0` 可以关闭这个功能。
```
Comment (TomShawn) [inline statistics.md:111]:
```suggestion
在查询语句执行时，TiDB 会以 `feedback-probability` 的概率收集反馈信息，并将其用于更新直方图和 Count-Min Sketch。`feedback-probability` 可通过配置文件修改，其默认值是 `0.05`。设置成 `0.0` 可以关闭这个功能。
```

**Comment by TomShawn** [comment]
@winoros 需要改 v3.1/v3.0/v2.1 吗？这三个版本也有一样的描述。

**Review by SunRunAway — COMMENTED** [review]
LGTM

File: statistics.md [hunk 3]
```diff
@@ -126,7 +126,11 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
 
 当某个表 `tbl` 的修改行数与总行数的比值大于 `tidb_auto_analyze_ratio`，并且当前时间在 `tidb_auto_analyze_start_time` 和 `tidb_auto_analyze_end_time` 之间时，TiDB 会在后台执行 `ANALYZE TABLE tbl` 语句自动更新这个表的统计信息。
 
-在查询语句执行时，TiDB 会以 `feedback-probability` 的概率收集反馈信息，并将其用于更新直方图和 Count-Min Sketch。`feedback-probability` 可通过配置文件修改，其默认值是 `0.0`。
+在查询语句执行时，TiDB 会以 `feedback-probability` 的概率收集反馈信息，并将其用于更新直方图和 Count-Min Sketch。`feedback-probability` 可通过配置文件修改，其默认值是 `0.05`。设置成 `0.0` 可以关闭这个功能。
```
Comment (TomShawn) [inline statistics.md:129]:
```suggestion
在查询语句执行时，TiDB 会以 `feedback-probability` 的概率收集反馈信息，并将其用于更新直方图和 Count-Min Sketch。可通过配置文件修改 `feedback-probability`，其默认值是 `0.05`。设置成 `0.0` 可以关闭这个功能。
```

File: statistics.md [hunk 4]
```diff
@@ -126,7 +126,11 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
...
+
+> **注意：**
+>
+> 在配置文件中如果将 `feedback-probability` 设置为 `0` 会导致设置失败报错，需要设置成 `0.0`，才可以将 `feedback-probability` 关闭。
```
Comment (TomShawn) [inline statistics.md:133]:
```suggestion
> 在配置文件中如果将 `feedback-probability` 设置为 `0` 会导致设置失败并报错。需要设置成 `0.0` 才可以关闭 `feedback-probability`。
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**